### PR TITLE
Update fmt.sh to check for binary.

### LINKED ISSUE #280: The fmt script should use gofumpt binary if it exists
Create an if branch like [lint](https://github.com/alpstable/gidari/blob/main/scripts/lint.sh#L4) that checks to see if [gofumpt](https://github.com/mvdan/gofumpt) exists and if it does do this:

```
gofumpt -l -w .
```

### PR DESCRIPTION
**Pull request by SkyWolf369**
Resolves #280

### CODE REVIEW

**Comment by SkyWolf369** [comment]
@prestonvasquez Should solve the issue!

File: scripts/fmt.sh [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+	gofumpt -l -w .
+else
+
 docker-compose -f "docker-compose.yml" up -d \
     --remove-orphans \
     --force-recreate \
     --build fmt
 
 docker-compose -f docker-compose.yml run --rm fmt
+
```
Comment (prestonvasquez) [inline scripts/fmt.sh:14]: Can you remove this empty line?

**Review by prestonvasquez — APPROVED** [review]

File: scripts/fmt.sh [hunk 2]
```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+if hash gofumpt 2>/dev/null;
+then
+	gofumpt -l -w .
+else
+
```
Comment (prestonvasquez) [inline scripts/fmt.sh:7]: Also remove this empty line

**Review by prestonvasquez — CHANGES_REQUESTED** [review]

**Comment by SkyWolf369** [comment]
Good?

**Comment by prestonvasquez** [comment]
@SkyWolf369 Well done! Thank you for the contribution :)

**Pull request merged**